docs/src/main/sphinx/release/release-362.md

```diff
@@ -0,0 +1,58 @@
+* Fix `ALTER SCHEMA ... SET AUTHORIZATION` to resolve roles using the catalog of the schema instead of the session catalog. ({issue}`9066`)
+* Fix query failure when query contains a cast from `varchar` to a shorter `char`. ({issue}`9036`)
+* Fix planning of recursive queries when the recursion the base plan or the recursion step plan produce duplicate outputs.
+* Split metrics that are reported in operator stats into `connectorMetrics` and `metrics`. ({issue}`9156`)
+
+## Security
+
+* Add `IN catalog` clause to `CREATE ROLE`, `DROP ROLE`, `GRANT ROLE`, `REVOKE ROLE`, and `SET ROLE` to specify 
+  the target catalog of the statement instead of using the current session catalog. This change is necessary to 
+  support system roles in the future. The old behavior of these statements can be restored by setting the 
+  `deprecated.legacy-catalog-roles` config property. ({issue}`9087`)
+
+## BigQuery connector
+
+* Support case insensitive name matching for BigQuery views. ({issue}`9164`)
+* Change type mapping of BigQuery `datetime` from `timestamp(3)` to `timestamp(6)` in Trino. ({issue}`9052`)
+* Change type mapping of BigQuery `time` from `time with time zone` to `time(6)` in Trino. ({issue}`9052`)
+* Change type mapping of BigQuery `timestamp` from `timestamp(3) with time zone` to `timestamp(6) with time zone` in Trino. ({issue}`9052`)
+* Fix query failure for predicate pushdown to BigQuery `datetime` type. ({issue}`9005`)
+* Fix incorrect results when using parameterized `numeric` type with non-default precision and scale. ({issue}`9060`)
+* Fix query failure when accessing tables having unsupported type columns. ({issue}`9086`)
+* Fix query failure for predicate pushdown to BigQuery `float64` type with `+infinity` or `-infinity` value. ({issue}`9122`)
+
+## Cassandra connector
+
+* Change minimum number of speculative executions from 2 to 1. ({issue}`9096`)
+
+## Hive connector
+
+* Limit maximum file size to 1 GB by default.  The limit is not exact and is performed best effort. The limit can be set with 
+  the `target_max_file_size` session property, or the `hive.target-max-file-size` configuration property. ({issue}`7991`)
+* Fix file-based metastore to always add the admin role by default. ({issue}`9066`)
+* Fix failure when writing Parquet files with Snappy compression on ARM64. ({issue}`9148`)
+* Fix performance regression where reading Parquet files that contain dictionaries. ({issue}`9161`)
+
+## Iceberg connector
+
+* Fix the mapping of nested fields between table metadata and Parquet file metadata. This
+  enables evolution of `row` typed columns for Iceberg tables stored in Parquet. ({issue}`9124`)
+
+## Pinot connector
+
+* Support pushdown of filters on `VARBINARY` columns to Pinot. ({issue}`9180`)
+* Fix handling of `IN` and `NOT IN` clauses for `VARCHAR` columns in presence of aggregations. ({issue}`9133`)
```

Comment:
I feel "Fix handling" is little unclear. It was incorrect result issue?

Reply:
Yes. I think it was a correctness issue (but pre-existing).

cc: @elonazoulay can you clarify once?